Comparison/src/cz/fidentis/comparison/procrustes/ProcrustesAnalysis.java

```diff
@@ -542,6 +542,59 @@ public ICPTransformation rotate(ProcrustesAnalysis pa2) {
         
         return new ICPTransformation(null, 1.0f, null, 0.0f, r);
     }
+    
+    /**
+     * Method for PDM computation, works same as normal rotate
+     * http://en.wikipedia.org/wiki/Orthogonal_Procrustes_problem
+     *
+     * @param pa2 another configuration
+     */
+    public ICPTransformation rotatePDM(ProcrustesAnalysis pa2) {
+        Matrix transConf2;
+        Matrix origConf1;
+        Matrix landmarkConf;
+        Matrix svdMat;
+        Matrix u;
+        Matrix v;
+        Matrix r;
+        Matrix transU;
+        List<Integer> cor = getFPtypeCorrespondence(pa2);
+        
+        /// VYTVORENIE LISTU INTOV LANDMARKOV
+        List<Integer> cor2 = getFPtypeCorrespondence(this);

```

Comment:
preco vytvaras korespondenciu medzi rovnakou konfiguraciou? Pokial ju potrebujes ako List<int>, tak rovno ziskaj zoznam klucov z premennej config, bude to rychlejsie a bude to davat vacsi zmysel